builds/e2e/connectivity.yaml

```diff
@@ -401,4 +401,4 @@ jobs:
           customEdgeHub.image: '$(customEdgeHub.image)'
```

Comment:
The test you posted ran very short:
https://dev.azure.com/msazure/One/_build/results?buildId=47404062&view=logs&j=2d6c4e75-f9d5-559b-a1b6-bfd270ae0f7e&t=54fdf04c-4d14-5ee6-4214-247e6ff742d8

Is this expected?